### PR TITLE
feat: add gemini code sample and test for text-only input

### PR DESCRIPTION
**Pull request by irataxy**
Fixes b:336785030

Please merge on approval.

### CODE REVIEW

File: aiplatform/api/AIPlatform.Samples/TextInputSample.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        };
+        content.Parts.AddRange(new List<Part>()
+        {
+            new()
+            {
+                Text = prompt
+            }
+        });
+
+        var generateContentRequest = new GenerateContentRequest
```
Comment (jskeet) [inline aiplatform/api/AIPlatform.Samples/TextInputSample.cs:61]: Please see other recently-merged PRs for clearer ways of constructing requests - it's better to do it all in a single statement, as that makes the structure of the request much clearer.

Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/TextInputSample.cs:61]: +1. This is basically what you need to do instead:

```
        var generateContentRequest = new GenerateContentRequest
        {
            Model = $"projects/{projectId}/locations/{location}/publishers/{publisher}/models/{model}",
            Contents =
            {
                new Content
                {
                    Role = "USER",
                    Parts =
                    {
                        new Part { Text = prompt }
                    }
                }
            }
        };
```

Comment (irataxy) [reply aiplatform/api/AIPlatform.Samples/TextInputSample.cs]: Done

File: aiplatform/api/AIPlatform.Samples.Tests/AIPlatform.Samples.Tests.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```
Comment (meteatamel) [inline aiplatform/api/AIPlatform.Samples.Tests/AIPlatform.Samples.Tests.csproj:3]: Do you really need net8.0?

Comment (irataxy) [reply aiplatform/api/AIPlatform.Samples.Tests/AIPlatform.Samples.Tests.csproj]: reverted

**Review by gericdong — APPROVED** [review]

**Review by amanda-tarafa — APPROVED** [review]

**Pull request merged**